test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -51,17 +51,17 @@ public async Task GivenFhirServer_WhenDataIsExported_ThenExportedDataIsSameAsDat
             IList<Uri> blobUris = await CheckExportStatus(contentLocation);
 
             // Download exported data from storage account
-            Dictionary<string, string> dataFromExport = await DownloadBlobAndParse(blobUris);
+            Dictionary<(string resourceType, string resourceId), string> dataFromExport = await DownloadBlobAndParse(blobUris);
```

Comment:
Changes is here are unrelated, but these tests were failing when my database had resources with different types but the same id.